src/widgets/input-filter-with-criteria/components/input-filter-with-criteria.jsx

```diff
@@ -13,74 +13,74 @@ const {
   BUTTON_SEARCH_CLASS,
 } = WIDGET_INPUT_FILTER_WITH_CRITERIA;
 
+const InputFilterWithCriteria = props => {
+
+  const { 
+    typeItem,
+    criteriaValues,
+    label 
+  } = props;
+
+  useEffect(() => {
+    if (props.loadOnInit)
+      props.loadSearchResult(props.typeItem);
+  }, []);
```

Comment:
Are you sure this dependencies are correct ? I think you depend of props.loadOnInit and props.typeItem. 
You should install the eslint plugin dedicated to hooks it will help you.